pfunk/_repotools.py

```diff
@@ -0,0 +1,14 @@
+#
+# Pints functional testing module.
+#
+# This file is part of Pints Functional Testing.
+#  Copyright (c) 2017-2018, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the Pints
+#  functional testing software package.
+#
+
+import time
+
+
+def format_date(seconds_since_epoch):
```

Comment:
Why did this get its own module?

Reply:
Could go in `_util` for example

Reply:
I probably didn't notice that file :-/. I've moved it.